client/my-sites/theme/main.jsx

```diff
@@ -1016,14 +1018,25 @@ class ThemeSheet extends Component {
 	getBannerUpsellTitle = () => {
 		const {
 			isBundledSoftwareSet,
+			themeSoftwareSet,
 			isExternallyManagedTheme,
 			translate,
 			isSiteEligibleForManagedExternalThemes,
 			isMarketplaceThemeSubscribed,
 		} = this.props;
 
 		if ( isBundledSoftwareSet && ! isExternallyManagedTheme ) {
-			return translate( 'Access this WooCommerce theme with a Business plan!' );
+			const themeSoftware = themeSoftwareSet[ 0 ];
+
+			if ( ! bundleSettings[ themeSoftware ] ) {
+				return translate( 'Access this theme with a Business plan!' );
+			}
+
+			const bundleName = bundleSettings[ themeSoftware ].name;
+
+			return translate( 'Access this %(bundleName)s theme with a Business plan!', {
+				args: { bundleName },
+			} );
```

Comment:
Please add a translators comment to make it easier to translate this effectively.

Reply:
Sure! Added in https://github.com/Automattic/wp-calypso/pull/84483/commits/707dffc27cde6ca5852c39af4746d6466bd58a78